apps/nextjs/src/stores/moderationStore/actionFunctions/handleToxicModeration.tsx

```diff
@@ -9,9 +9,9 @@ export const handleToxicModeration = (
   get: () => ModerationStore,
 ) => {
   set({ toxicModeration: mod });
-  const { chatActions } = get();
+  const { chatActions, lessonPlanActions } = get();
   invariant(chatActions, "Passed into store in provider");
+  invariant(lessonPlanActions, "Passed into store in provider");
   chatActions.setMessages([], false);
-
-  //  @TODO setOverrideLessonPlan({});
+  lessonPlanActions.resetStore();
```

Comment:
I think we should decide on a pattern for actions between stores. Should the moderation store be telling the lesson plan store what to do (lessonPlanActions.resetStore), or should it tell the store the situation for it to react to (lessonPlanActions.handleToxicModeration)

Reply:
AI-830